.github/actions/clickhouse/action.yml

```diff
@@ -17,6 +17,12 @@ runs:
     - name: Set up Docker
       uses: docker-practice/actions-setup-docker@master
 
+    - name: Install Fdb C Lib
+      shell: bash
+      run: |
+        wget https://github.com/apple/foundationdb/releases/download/6.3.24/foundationdb-clients_6.3.24-1_amd64.deb
```

Comment:
The `6.3.24` package is broken. I suggest using either `6.3.23` or `7.1.x`. See this FDB issue: https://github.com/apple/foundationdb/issues/6872

Reply:
Got it, thank you for the reminding.